piksi_multi_cpp/include/piksi_multi_cpp/sbp_callback_handler/sbp_callback_handler_relay/sbp_callback_handler_relay_logging.h

```diff
@@ -0,0 +1,29 @@
+#ifndef PIKSI_MULTI_CPP_SBP_CALLBACK_HANDLER_SBP_CALLBACK_HANDLER_RELAY_LOGGING_H_
+#define PIKSI_MULTI_CPP_SBP_CALLBACK_HANDLER_SBP_CALLBACK_HANDLER_RELAY_LOGGING_H_
+
+#include <libsbp/logging.h>
+#include <piksi_multi_msgs/Fwd.h>
+#include <piksi_multi_msgs/Log.h>
+#include "piksi_multi_cpp/sbp_callback_handler/sbp_callback_handler_relay/sbp_callback_handler_relay.h"
+
+namespace piksi_multi_cpp {
+
+class SBPCallbackHandlerRelayLog
+    : public SBPCallbackHandlerRelay<msg_log_t, piksi_multi_msgs::Log> {
+ public:
+  inline SBPCallbackHandlerRelayLog(const ros::NodeHandle& nh,
+                                    const std::shared_ptr<sbp_state_t>& state)
+      : SBPCallbackHandlerRelay(nh, SBP_MSG_LOG, state, "log") {}
+};
+
+class SBPCallbackHandlerRelayFwd
```

Comment:
hmm I think for these a macro could be justified - such that you only get a list of instantiations such as

CreateHandlerRelay(msg_fwd_t, pksi_multi_msgs.:Fwd, SBP_MSG_FWD, "fwd")